workspaces/ai-integrations/plugins/catalog-backend-module-model-catalog/src/clients/ModelCatalogGenerator.ts

```diff
@@ -189,7 +189,36 @@ export function GenerateModelServerAPI(
   };
 
   if (api.tags !== undefined) {
-    modelServerAPIEntity.metadata.tags = api.tags;
+    modelServerAPIEntity.metadata.tags = sanitizeTags(api.tags);
   }
   return modelServerAPIEntity;
 }
+
+function sanitizeMetadataName(modelName: string): string {
```

Comment:
yeah for all these sanitizes, except for replacing a space with an empty char i.e. just remove the space I think there is slippery slope here wrt replacing spaces with dashes even perhaps

WDYT

Reply:
I generally prefer dashes as it improves readability, but it's not a strong preference. 

Just to clarify, you're suggested we instead replace spaces with an empty char?

Reply:
yes empty char .... so `Version 1` becomes `Version1`

Reply:
Updated